ada/ast.py

```diff
@@ -11912,6 +11912,23 @@ def xref_equation():
         )
 
 
+class CallExprKind(Enum):
+    """
+    Kind of CallExpr type.
+
+    - ``NoCallExpr`` is the null value for this enum.
+    - ``Call`` is when the CallExpr is a procedure or function call.
+    - ``ArraySlice``, ``ArrayIndex`` is when the CallExpr is in fact an array
+      slice or an array subcomponent access expression, respectively.
+    - ``TypeConversion`` is when the CallExpr is a type conversion.
+    """
+    NoCallExpr = EnumValue(is_default=True)
```

Comment:
Is it necessary to have the `NoCallExpr` value? Maybe we should raise an exception instead?